File: mysql/jython/read/mysql_read.py
#! /usr/bin/jython
# -*- coding: utf-8 -*-
#
#	mysql_read.py
#					Jan/26/2011
#
# ----------------------------------------------------------------
#
import sys
from java.lang import	System
#
import java
from java.sql import DriverManager
#
sys.path.append ('/var/www/data_base/common/jython_common')
from jython_rdb_manipulate import display_proc
# ----------------------------------------------------------------
System.out.println ("*** start ***")
print ("*** 開始 ***")

#
url="jdbc:mysql://host_mysql/city"
user = "scott"
password = "tiger"

conn = DriverManager.getConnection (url,user, password)

display_proc (conn)

conn.close()
print ("*** 終了 ***")
#
# ----------------------------------------------------------------

File: mysql/jruby/read/mysql_read.rb
#! /usr/bin/jruby
# -*- coding: utf-8 -*-
#
#	mysql_read.rb
#					Jul/15/2014
#
# --------------------------------------------------------
include Java
import java.sql.DriverManager
import java.lang.System
#
load '/var/www/data_base/common/jruby_common/jruby_sql_manipulate.rb'
# --------------------------------------------------------
puts "*** 開始 ***"
str_connect = "jdbc:mysql://host_mysql/city"
conn = DriverManager.getConnection(str_connect,"scott", "tiger")

sql_read_proc(conn)

conn.close()
puts "*** 終了 ***"
# --------------------------------------------------------
